client/src/components/Input.js

```diff
@@ -1,12 +1,25 @@
 import styled from '@emotion/styled';
 import colors from '../utils/colorsUtils';
 
+const inputSize = {
+  large: { padding: '10px', margin: '7px' },
+};
```

Comment:
Overall good solution 👍 

Right now, there is only one size for the input (large) but I assume you will add different sizes in the future.

Reply:
Thanks for the review!
You are right, for now there is only the large input, later I will add more sizes.
For now, I plan to have the same font-size everywhere, so I don't think I will need to adjust it in this project, but you are very right in general and probably I will come to a point, where I decide not to have the same font-size everywhere. 
You are also right about the margin. Got it in the first place, to have space between two inputs. But this is actually not recommendable. I think it's much better to have no margin for the input and get the margin over a parent flex-box container. This will make the input element more variable.